### DISCUSSION TITLE
Plugin API

### DISCUSSION THREAD

**Discussion by JustSteveKing** [minicli/minicli#80] [Ideas]
Should we investigate creating a wrapper around the composer plugin API so that people can create their own plugins for minicli?

**Comment by WendellAdriel** [comment]
I'd say that we can do that because this way we can have more contributors and more tools added to the minicli ecosystem

**Comment by JustSteveKing** [comment]
This has now been created: https://github.com/minicli/mini-plugin

**Comment by WendellAdriel** [comment]
I'm closing this one since this was already created
